src/app/detail/ui/ToggleController.tsx

```diff
@@ -8,9 +8,9 @@ import { useSelector } from 'react-redux'
 
 export const ToggleController = ({ children }: { children: ReactNode }) => {
   const { showSidebar } = useSelector(selectTaskDetails)
-  const matches = useMediaQuery('(max-width:600px)')
+  const matches = useMediaQuery('(max-width:700px)')
 
-  const nonMobile = useMediaQuery('(min-width:600px)')
+  const nonMobile = useMediaQuery('(min-width:700px)')
```

Comment:
700px assumption is out of sync with the rest of the Copilot app implementation.